<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Easy Loader</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>3</number>
      </property>
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
      <property name="movable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Routnig Map</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Dictionary</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="Editor_Tab">
       <attribute name="title">
        <string>Config Editor</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="Loader_Tab">
       <attribute name="title">
        <string>Loader</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>22</height>
    </rect>
   </property>
   <property name="defaultUp">
    <bool>true</bool>
   </property>
   <property name="nativeMenuBar">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menuHello">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionPreferences"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuSystem">
    <property name="title">
     <string>System</string>
    </property>
    <addaction name="actionConfiguration_Wizard"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionLoader"/>
    <addaction name="actionEditor"/>
    <addaction name="actionDictionary"/>
    <addaction name="actionConfig_Editor"/>
   </widget>
   <addaction name="menuHello"/>
   <addaction name="menuSystem"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionConfiguration_Wizard">
   <property name="text">
    <string>Configuration Wizard</string>
   </property>
  </action>
  <action name="actionEditor">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Routing Map</string>
   </property>
  </action>
  <action name="actionLoader">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Loader</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="text">
    <string>Preferences</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionDictionary">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Dictionary</string>
   </property>
  </action>
  <action name="actionConfig_Editor">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Config Editor</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
